evalml/pipelines/components/estimators/regressors/arima_regressor.py

```diff
@@ -170,14 +144,22 @@ def fit(self, X, y=None):
         Raises:
             ValueError: If X was passed to `fit` but not passed in `predict`.
         """
+        X, y = self._manage_woodwork(X, y)
         if y is None:
             raise ValueError("ARIMA Regressor requires y as input.")
+        if X is not None:
+            target_leakage = TargetLeakageDataCheck().validate(X, y)["actions"]
+            leaked_columns = (
+                target_leakage[0]["metadata"]["columns"] if target_leakage else []
+            )
```

Comment:
These nested list/dict structures :(. I think we might want to think again about the output of `.validate()` to make accessing things like the target leaked columns clearer.  I'll file an issue for this.